book/animations.md

```diff
@@ -2073,35 +2081,36 @@ testing is now complete. You should now be able to render
 
 The blue square should now be underneath the green square, so overlap
 testing is now complete. You should now be able to render
-this example correctly:
+[this example](https://browser.engineering/examples/example13-transform-overlap.html)
+correctly:
 
 ::: {.transclude .html}
 www/examples/example13-transform-overlap.html
 :::
 
 :::
 
-Which should look like this:
+It should look like Figure 7.
 
 ::: {.center}
-![Example of transformed overlap, clipping and blending](examples/example13-transform-overlap.png)
+![Figure 7: Example of transformed overlap, clipping and blending](examples/example13-transform-overlap.png)
 :::
 
 Notice how this example exhibits *two* interesting features we had
 to get right when implementing compositing:
 
 * Overlap testing (without it, the elements would paint in the wrong order);
-if this code were missing it would incorrectly render like this:
+if this code were missing it would incorrectly render like Figure 8.
 
 ::: {.center}
-![Wrong rendering because overlap testing is missing](examples/example13-transform-overlap-wrong1.png)
+![Figure 8: Wrong rendering because overlap testing is missing](examples/example13-transform-overlap-wrong1.png)
 :::
 
 * Reusing cloned effects (without it, blending and clipping would be wrong);
-if this code were missing it would incorrectly render like this:
+if this code were missing it would incorrectly render like Figure 9.
 
 ::: {.center}
-![Wrong rendering because of incorrect blending](examples/example13-transform-overlap-wrong2.png)
+![Figure 9: Wrong rendering because of incorrect blending](examples/example13-transform-overlap-wrong2.png)
```

Comment:
For typesetting it would be nice to put this series of images in a single figure with subfigures. But that's not for right now.